.github/workflows/hermetic_library_generation.yaml

```diff
@@ -15,7 +15,9 @@
 # downstream client libraries before they are released.
 name: Hermetic library generation upon generation config change through pull requests
 on:
-  pull_request:
+  # TODO: uncomment this before merging to prevent a formatting regression
```

Comment:
This need to be uncommented now?

Reply:
I will uncomment it right after restoring the formatted files. I'll cancel the job once the CIs start.